server/src/services/ScuntTeamServices.js

```diff
@@ -319,15 +319,77 @@ const ScuntTeamServices = {
                   if (err) {
                     reject(err);
                   } else {
+                    var scuntTeamDict = {};
```

Comment:
```suggestion
                    let scuntTeamDict = {};
```
This should probably be `let` instead of `var` because of scoping issues with `var`

Reply:
Resolved in https://github.com/UofT-Frosh-Orientation/orientation-website/pull/573/commits/39194d4cfb1f2fecf14c82e5d4eaee1ff65175aa#diff-8cc001f2e5cfb7ab71cd61b96c1a5a41d2d069b01a47679f12a9af561a235b0bR322